doc/source/rllib/getting-started.rst

```diff
@@ -0,0 +1,449 @@
+        num_epochs=10,
+    )
+
+Finally, you build the actual :py:class:`~ray.rllib.algorithms.algorithm.Algorithm` instance
+through calling your config's :py:meth:`~ray.rllib.algorithm.algorithm_config.AlgorithmConfig.build_algo`
+method.
+
+.. testcode::
+
+    # Build the Algorithm (PPO).
+    ppo = config.build_algo()
+
+
+.. note::
+
+    See here to learn about the :ref:`methods you can use to configure your Algorithm <rllib-algo-configuration-docs>`.
+
+
+Run the algorithm
++++++++++++++++++
+
+After you built your :ref:`PPO <ppo>` from its configuration, you can ``train`` it for a number of
+iterations through calling the :py:meth:`~ray.rllib.algorithms.algorithm.Algorithm.train` method,
+which returns a result dictionary that you can pretty-print for debugging purposes:
+
+.. testcode::
+
+    from pprint import pprint
+
+    for _ in range(5):
+        pprint(ppo.train())
+
+
+Checkpoint the algorithm
+++++++++++++++++++++++++
+
+To save the current state of your :py:class:`~ray.rllib.algorithms.algorithm.Algorithm`,
+create a ``checkpoint`` through calling its :py:meth:`~ray.rllib.algorithms.algorithm.Algorithm.save_to_path` method,
+which returns the directory of the saved checkpoint.
+
+Instead of not passing any arguments to this call and letting the algorithm decide where to save
+the checkpoint, you can also provide a checkpoint directory yourself:
+
+.. testcode::
+
+    checkpoint_path = ppo.save_to_path()
+
+    # OR:
+    # ppo.save_to_path([a checkpoint location of your choice])
+
+
+Evaluate the algorithm
+++++++++++++++++++++++
+
+RLlib supports setting up a separate :py:class:`~ray.rllib.env.env_runner_group.EnvRunnerGroup`
+for the sole purpose of evaluating your model from time to time on the RL environment.
+
+Use your config's :py:meth:`~ray.rllib.algorithms.algorithm_config.AlgorithmConfig.evaluation` method
+to set up the details. By default, RLlib doesn't perform evaluation during training and only reports the
+results of collecting training samples with its "regular" :py:class:`~ray.rllib.env.env_runner_group.EnvRunnerGroup`.
+
+
+.. testcode::
+    :hide:
+
+    ppo.stop()
+
+
+.. testcode::
+
+    config.evaluation(
+        # Run one evaluation round every iteration.
+        evaluation_interval=1,
+
+        # Create 2 eval EnvRunners in the extra EnvRunnerGroup.
+        evaluation_num_env_runners=2,
+
+        # Run evaluation for exactly 10 episodes. Note that because you have
+        # 2 EnvRunners, each one runs through 5 episodes.
+        evaluation_duration_unit="episodes",
+        evaluation_duration=10,
+    )
+
+    # Rebuild the PPO, but with the extra evaluation EnvRunnerGroup
+    ppo_with_evaluation = config.build()
+
+    for _ in range(3):
+        pprint(ppo_with_evaluation.train())
+
+
+.. _rllib-with-ray-tune:
+
+RLlib with Ray Tune
++++++++++++++++++++
+
+All RLlib :py:class:`~ray.rllib.algorithms.algorithm.Algorithm` classes are compatible with
```

Comment:
That's not true: All offline RL classes in the new stack do not work with Ray Tune, yet.

Reply:
done